std_msgs/Header header
geometry_msgs/Point logo_pos
geometry_msgs/Point QR_pos
bool logo_update
bool qr_update

